libs/shared/methodologies/ai-attachment-validator/src/ai-attachment-validator.service.spec.ts

```
@@ -0,0 +1,299 @@
+      expect(mappedDto).toEqual({
+        attachmentPaths: [dto.attachmentPath],
+        context: dto.additionalContext,
+        documentJson: dto.document,
+        mode: VALIDATION_MODE,
+      });
+    });
+
+    it('should correctly map the DTO without additional context', () => {
+      const dto = stubAiValidateAttachmentDto();
+
+      delete dto.additionalContext;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)